src/embed/init.impl.ts

```diff
@@ -104,11 +106,17 @@ export default function _dlo_initializeFromWindow() {
 
     // Logging must be initialized before telemetry since telemetry errors may be logged
     if (win._dlo_telemetryProvider) {
-      Telemetry.setInstance(win._dlo_telemetryProvider);
+      Telemetry.setProvider(win._dlo_telemetryProvider);
     } else {
-      Telemetry.setInstance(Telemetry.withExporter(win._dlo_telemetryExporter));
+      Telemetry.setProvider(
+        Telemetry
+          .withExporter(win._dlo_telemetryExporter)
```

Comment:
Would it be a good idea to add a comment here about what the expected `Telemetry` metric collection behavior is if no `win._dlo_telemetryExporter` has been defined?